stream/stream_bstr.c

```diff
@@ -44,12 +44,18 @@ static int64_t get_size(stream_t *s)
     return p->data.len;
 }
 
+static void close(struct stream *s)
```

Comment:
This name evidently collides with something in mingw. Maybe `bstr_close` or something instead.

Reply:
Renamed the private functions and also dropped to unneeded included that led to the conflict in the first place.